src/checker.rs

```diff
@@ -12,7 +17,7 @@ pub fn check(rules: &Rules, raw: &str) -> bool {
                 .nth(1)
                 .map(|c| !c.is_alphabetic())
                 .unwrap_or_default()
-        || trimmed.chars().filter(|c| c.is_alphabetic()).count() < rules.min_characters
+        || !in_limit(trimmed.chars().filter(|c| c.is_alphabetic()).count(), rules.min_characters, rules.max_characters)
```

Comment:
To make this line a bit shorter, what do you think about assigning the count to a variable before this fill check?

Reply:
Yep, better...

Reply:
Done...